Formula/traefik.rb

```diff
@@ -0,0 +1,61 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "Træfik, a modern reverse proxy"
+  homepage "https://traefik.io/"
+  url "https://github.com/containous/traefik/releases/download/v1.3.8/traefik-v1.3.8.src.tar.gz"
```

Comment:
needs an explicit
```
  version "1.3.8"
```
because the version detected from that url is `1.3.8.src` not `1.3.8`

Reply:
Done.